scipy/sparse/_construct.py

```diff
@@ -241,6 +252,12 @@ def diags(diagonals, offsets=0, shape=None, format=None, dtype=None):
     dtype : dtype, optional
         Data type of the matrix.
 
+    Returns
+    -------
+    new_matrix : dia_matrix
+        dia_matrix holding the values in `diagonals` offset from the main diagonal
```

Comment:
Double backticks here and the next line

Reply:
I think single backticks here are correct since these are arguments to the function

Reply:
Yes, it looks like single ticks should be used for arguments so they get italics. Thanks!